### PR TITLE
Add Code Highlighting to RemoteContent

### PR DESCRIPTION
**Pull request by alex-fusionauth**
Prior to change
localhost_3000_docs_sdks_react-sdk

After Change
localhost_3000_docs_sdks_react-sdk (1)

### CODE REVIEW

File: astro/src/components/RemoteContent.astro [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 /**
-  * Remote content fetcher. Remote content is assumed to be markdown.
-  */
-import { marked } from 'marked';
+ * Remote content fetcher. Remote content is assumed to be markdown.
+ */
+import { Markdown } from "astro-remote";
```
Comment (lyleschemmerling) [inline astro/src/components/RemoteContent.astro:5]: This is not found, did you include this in the package.json?

**Review by lyleschemmerling — CHANGES_REQUESTED** [review]

File: astro/src/components/RemoteContent.astro (hunk 1 above)
Comment (alex-fusionauth) [reply astro/src/components/RemoteContent.astro:5]: my bad @lyleschemmerling I think I installed before I created the new branch.

**Comment by lyleschemmerling** [comment]
My concern with this is that this will be at least the third way of parsing markdown into HTML that we have in the site code which is making me a little uneasy. In theory we can do this with what we have already, it just means mucking around in tools yet.

And now that I am looking at this I realize that the `RemoteContent` component will fail silently if the source is not available, which I also think needs to change because if we can't fetch the content then we shouldn't build.

**Review by synedra — APPROVED** [review]
LGTM

**Review by synedra — APPROVED** [review]
LGTM

**Review by lyleschemmerling — APPROVED** [review]

**Pull request merged**